css/build.css

```diff
@@ -282,4 +282,11 @@ input.form-control {
     margin: 0 auto;
     max-width: 100%;
   }
+
+  [data-widget-package="com.fliplet.login-data-source"] .btn.btn-primary,
+  [data-widget-package="com.fliplet.login-data-source"] .btn.btn-secondary,
+  [data-widget-package="com.fliplet.login-data-source"] .btn.btn-danger {
+    min-width: 30% !important;
```

Comment:
Why do we need to add new selectors/declarations?

Is there a reason we can't just:

1. Update the `width: 30%` that's already on the page to `min-width: 30%`
1. Change the existing selectors to be more specific to the component only

Reply:
Actually all button styles are the same so I merged them into one. Is adding data-widget-package as a selector is fine?